yellowbrick/contrib/classifier/boundaries.py

```diff
@@ -202,7 +202,7 @@ def __init__(self,
 
         self.x = x
         self.y = y
-        self.features_ = features
```

Comment:
I'm wondering if this should be `feature_labels_` in many cases -- I'll have a larger comment for discussion towards the end.